dotnet-interactive.IntegrationTests/LoggingTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.App.IntegrationTests.Utility;
+using Microsoft.DotNet.Interactive.Utility;
+using Xunit;
+
+namespace Microsoft.DotNet.Interactive.App.IntegrationTests
+{
+    public class LoggingTests
+    {
+        [Fact]
```

Comment:
So that these won't run as part of the normal dev loop, let's use a derived `FactAttribute` that skips unless, e.g., it's a release build or it detects that we're running in a CI build.

Reply:
Added a commit with exactly this.  I confirmed locally that the VS Test Explorer excluded this test.  I also added a doc comment explaining that to run it locally the `[IntegrationTest]` attribute will need to be replaced with `[Fact]`.

I'll confirm that the test still runs during the integration leg and that the `.trx` file was appropriately added to the test run.